src/main/twirl/gitbucket/core/settings/edithooks.scala.html

```diff
@@ -30,6 +30,16 @@
               }
               <button class="btn btn-default" id="test">Test Hook</button>
             </fieldset>
+            <fieldset class="form-group">
+              <label class="strong">Security Token</label>
+              <div></div>
+              @if(create){
+              <input type="text" name="token" id="token" placeholder="leave blank for no X-Hub-Signature usage" value="@webHook.token" class="form-control" style="display: inline; width: 500px; vertical-align: middle;" />
+              } else {
+              <input type="text" value="@webHook.token" class="form-control" style="display: inline; width: 500px; vertical-align: middle;" disabled />
```

Comment:
oh... I seted disabled to url field, because, 
gitbucket's url field is primary key of web_hook,
And I think it is too complex that unique-check, redirect when change url, validation error...
 ( In github, they created `id`, and use it for primary-key, and it make simple )

But token is not primary key, and I think it sould be editable. However , it may not now .